File: clients/Python/examples/MqAdminExample.py
from zbus import Broker, MqAdmin, Message
 
broker = Broker()
broker.add_tracker('localhost:15555')


admin = MqAdmin(broker)

msg = Message()
msg.topic = 'hong6'
res = admin.declare(msg)
for res_i in res:
    print(res_i)
 
broker.close()

File: clients/C/examples/MqAdminExample.cpp
#include "MqAdmin.h"  


int main_MqAdmin(int argc, char* argv[]) {  
	Logger::configDefaultLogger(0, LOG_INFO); 
	Logger* log = Logger::getLogger(); 


	Broker broker("localhost:15555");
	MqAdmin admin(&broker);
	std::vector<ServerInfo> res = admin.queryServer();

	string topic = "CPP_Topic";
	admin.declareTopic(topic); 
	admin.declareGroup(topic, "MyCpp");
	admin.removeTopic(topic);

	system("pause");
	return 0;
}